cypress/e2e/universal-search.test.ts

```diff
@@ -25,31 +25,42 @@ describe('Universal search bar', () => {
     cy.location('hash').should('equal', '#/tokens/ethereum/0x1f9840a85d5af5bf1d1762f925bdaddc4201f984')
   })
 
-  it('should go to the selected result when recent results are shown', () => {
-    // Search for uni token by name.
-    getSearchBar().type('uni')
-    cy.get('[data-cy="searchbar-token-row-UNI"]')
+  it(
```

Comment:
these changes should be in their own PR and not coupled to this hotfix

Reply:
it's failing this test though, this tests the same element we're fixing

Reply:
more context: i added this test when i implemented the SuspenseFallback stuff, basically this test was made for the same code were fixing here

Reply:
These are two different fixes.
1. Fixes the suspense issue causing a page crash
2. Fixes a flaky test relating to the searchbar that is currently failing on main
These should be their own PRs. The PR that fixes the flaky test by adding retries should not have the rider clause of altering use Suspense behaviour or vice versa because they touch the same component. They're two separate changes.

Reply:
The flaky test was actually added to cover the thing that's erroring. I'll split it out, but they are one and the same (test is testing the area that breaks, was added to cover the thing that broke). I do think adding yet another test to test backspace would be good here as well.